_includes/v23.1/backups/updated-backup-privileges.md

```diff
@@ -20,4 +20,17 @@ You can grant the `BACKUP` privilege to a user or role **without** the `SELECT`
 
 Members of the [`admin` role](security-reference/authorization.html#admin-role) can run all three types of backups (cluster, database, and table) without the need to grant a specific `BACKUP` privilege. However, we recommend using the `BACKUP` privilege model to create users or roles and grant them `BACKUP` privileges as necessary for stronger access control.
 
+### Privileges for managing a backup job
+
+To manage a backup job with [`PAUSE JOB`](pause-job.html), [`RESUME JOB`](resume-job.html), or [`CANCEL JOB`](cancel-job.html), users must:
```

Comment:
nit: Maybe say that users must have at least one of the following? - As opposed to having all of the following. Feel free to leave this as is if it's obvious.